### PR TITLE
dev-util/cutter: version bump to 1.8.1

### PR DESCRIPTION
**Pull request by winterheart**
dev-util/radare2-3.4.1 changed API which breakes compilation of cutter
(#684446). This breakage fixed in 1.8.1.

Closes: https://bugs.gentoo.org/684446

Signed-off-by: Azamat H. Hackimov azamat.hackimov@gmail.com
Package-Manager: Portage-2.3.62, Repoman-2.3.11

### CODE REVIEW

File: dev-util/cutter/cutter-1.8.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (juippis) [inline dev-util/cutter/cutter-1.8.1.ebuild]: Looking at eclasses, EAPI-7 should work here.

File: dev-util/cutter/cutter-1.8.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+	eqmake5 "${myqmakeargs[@]}" src
+}
+
+src_install() {
+	emake INSTALL_ROOT="${D}" install
+}
+
+pkg_postinst() {
+	xdg_desktop_database_update
+	gnome2_icon_cache_update
```
Comment (juippis) [inline dev-util/cutter/cutter-1.8.1.ebuild]: If EAPI-7 works, this is moved into xdg-utils.eclasS (that you already inherit) and is called `xdg_icon_cache_update` instead. Update this and `pkg_postrm` so you can drop gnome2-utils.eclass.
(@gyakovlev)

**Comment by juippis** [comment]
Also ping @otakuto, please review and test this :)

**Comment by otakuto** [comment]
Thank you @winterheart.
LGTM.
It works well.

**Comment by BinaryKhaos** [comment]
Please also note that the current release is v1.8.2 now.

**Comment by mgorny** [comment]
@winterheart, ping. Could you please update this to the newest version, also follow the review comments?

**Comment by winterheart** [comment]
Done

File: dev-util/cutter/cutter-1.8.2.ebuild [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+DEPEND="
+	${PYTHON_DEPS}
+	>=dev-qt/qtcore-5.9.1:5
+	>=dev-qt/qtgui-5.9.1:5
+	>=dev-qt/qtsvg-5.9.1:5
+	>=dev-qt/qtwidgets-5.9.1:5
+	dev-qt/qtnetwork:5
+	>=dev-util/radare2-3.5.1
+	jupyter? ( dev-python/jupyter )
```
Comment (mgorny) [inline dev-util/cutter/cutter-1.8.2.ebuild]: You sure you don't need PYTHON_USEDEP on that?

**Pull request closed without merging**